Mathlib/Combinatorics/SimpleGraph/Coloring.lean

```diff
@@ -297,8 +297,7 @@ theorem chromaticNumber_le_iff_colorable {n : ℕ} : G.chromaticNumber ≤ n ↔
   rw [Set.mem_setOf_eq] at this
   exact this.mono h
 
--- 2024-03-21
-@[deprecated Colorable.chromaticNumber_le]
+@[deprecated Colorable.chromaticNumber_le] -- 2024-03-21
 theorem chromaticNumber_le_card [Fintype α] (C : G.Coloring α) :
```

Comment:
Can't we use `since` here?

Reply:
Oh yes, we can: thanks for reminding me:
```suggestion
@[deprecated Colorable.chromaticNumber_le (since := "2024-03-21")]